server/db/schema/application.ts

```diff
@@ -132,7 +133,12 @@ export const applications = pgTable("application", {
 	customGitUrl: text("customGitUrl"),
 	customGitBranch: text("customGitBranch"),
 	customGitBuildPath: text("customGitBuildPath"),
-	customGitSSHKey: text("customGitSSHKey"),
+	customGitSSHKeyId: text("customGitSSHKeyId").references(
+		() => sshKeys.sshKeyId,
+		{
+			onDelete: "set null",
+		},
+	),
```

Comment:
to prevent the breaking change at the moment of generate the migration it gives two options:

1. Rename column 
2. New column

I think you selected the rename column which which for all the dokploy instances this will throw an error which will make the server cannot start, You need to first revert that migration and select `New Column` and drop the column `customGitSSHKey`  and obviously for all the instances who have value in this field will automatically delete, but in the release notes I will add this so they can handle safely and be aware, and this apply the same for `compose` table

Reply:
Thanks, I created new migrations. Currently, the SSH folder has 600, and it doesn't allow writing new keys. I think we should keep the SSH permission to 777 and create the SSH keys with 600. I'm testing this solution

Reply:
@Siumauricio https://github.com/Dokploy/dokploy/pull/276/commits/bda0689e18afdbc4127a260f7197c159b01a0a2b

it should be ok now